src/mmw/js/src/analyze/views.js

```diff
@@ -68,6 +72,14 @@ var AnalyzeWindow = Marionette.LayoutView.extend({
         }));
     },
 
+    showLoadingView: function() {
+        this.detailsRegion.show(new LoadingView());
```

Comment:
In accordance with @caseypt's comment on consolidating these views, consider using a pattern similar to one used in geocoding here: https://github.com/WikiWatershed/model-my-watershed/blob/develop/src/mmw/js/src/geocode/views.js#L13 and changing this to something along the lines of:

```javascript
showLoadingView: function() {
    this.detailsRegion.show(new StatusView(ICON_WORKING, MSG_WORKING));
}
```